http://www.cygwin.com/ml/libc-alpha/2011-09/msg00028.html

2011-09-06  Samuel Thibault  <samuel.thibault@ens-lyon.org>

* sysdeps/mach/hurd/sys/param.h (DEV_BSIZE): Define macro.

---
 sysdeps/mach/hurd/sys/param.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/sysdeps/mach/hurd/sys/param.h
+++ b/sysdeps/mach/hurd/sys/param.h
@@ -119,6 +119,10 @@
 #define	MAX(a,b) (((a)>(b))?(a):(b))
 
 
+/* Unit of `st_blocks'.  */
+#define DEV_BSIZE       512
+
+
 /* Scale factor for scaled integers used to count %cpu time and load avgs.
 
    The number of CPU `tick's that map to a unique `%age' can be expressed
